lib/tasks/sitemap.rake

```diff
@@ -0,0 +1,9 @@
+desc 'Notify Google of the updated sitemp'
+task :sitemap do
+    require 'net/http'
+    Net::HTTP.get(
+        'www.google.com',
+        '/ping?sitemap=' +
+        URI.escape('http://domain.com/sitemap.xml')
```

Comment:
Is `domain.com` correct or does it need to know the hostname somehow?

Reply:
No, that is just a placeholder, we need to provide either a variable or the actual domain name.

Reply:
Can it take it as an ENV VAR in the rake task than, eg

`bundle exec rake sitemap DOMAIN=https://my.domain.com`

Reply:
Could do, or

  URI.escape("#{app.root_url}" + "sitemap.xml")
    )
end
# app.root_url is "http://www.example.com/" for localhost. Can be configured in Application controller
# or environment via default_url_options

Reply:
It would only be needed for the production domain, I would have thought.